health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -73,10 +73,29 @@ const EditVillagePopulationPopUp = ({ onClose, census, onSuccess }) => {
       ...confirmedValues,
       [fieldKey]: value,
     });
-
+  
+    let error = !value || Number(value) <= 0 || Number(value) > 100000;
```

Biome

[error] This let declares a variable that is only assigned once.

'error' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)